hbase-client/src/main/java/org/apache/hadoop/hbase/client/MetricsConnection.java

```diff
@@ -466,6 +468,9 @@ public void updateRpc(MethodDescriptor method, Message param, CallStats stats) {
     if (callsPerServer > 0) {
       concurrentCallsPerServerHist.update(callsPerServer);
     }
+    // Update the counter that tracks RPCs by type.
+    final String methodName = method.getService().getName() + "_" + method.getName();
```

Comment:
Neat idea but it's coming in from some other work?

Reply:
Can we break out this change? Metrics are special. They are a separate dimension of compatibility. Having them in a separate commit allows them to get noticed. They should have their own release note. Etc.